projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionLogsUrlBuilder.java

```diff
@@ -50,6 +51,7 @@ public class JobExecutionLogsUrlBuilder {
   static final String EXECUTION_ID_VAR = "execution_id";
   static final String OP_ID_VAR = "op_id";
   static final String JOB_NAME_VAR = "job_name";
+  static final String TEAM_NAME_VAR = "team_name";
```

Comment:
Please update docs on config usages, for example https://github.com/vmware/versatile-data-kit/blob/main/projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml#L980

Reply:
Done! thanks